src/Benchmarks/Middleware/DebugInfoPageMiddleware.cs

```diff
@@ -42,6 +42,7 @@ public DebugInfoPageMiddleware(RequestDelegate next, IServerAddressesFeature ser
         public async Task Invoke(HttpContext httpContext)
         {
             httpContext.Response.ContentType = "text/html";
+            httpContext.Response.StatusCode = StatusCodes.Status404NotFound;
```

Comment:
This middleware is only used when none of the others match i.e. nothing was found.

Besides that, I mistakenly did a manual `wrk` run with an incorrect URL. Took much longer than it should have to realize why I was seeing higher-than-MVC numbers. This prevents future mistakes like that and browsers still show all the info.